pkg/rbac/common.go

```diff
@@ -178,7 +179,7 @@ func TypeFromContext(apiContext *types.APIContext, resource *types.RawResource)
 }
 
 func GetRTBLabel(objMeta metav1.ObjectMeta) string {
-	return objMeta.Namespace + "_" + objMeta.Name
+	return wranglerName.SafeConcatName(objMeta.Namespace, objMeta.Name)
```

Comment:
This is actually returning a different result than before, which means old labels will be invalid and will break current setups, and means [this test](https://github.com/rancher/rancher/blob/caba2c0622005c2cc63f5f6fe63f84a9533b2648/tests/framework/extensions/users/users.go#L194) breaks. Could we do this instead

```suggestion
	return wranglerName.SafeConcatName(objMeta.Namespace "_" + objMeta.Name)
```